Intersect.Server/Entities/Entity.cs

```diff
@@ -2724,8 +2724,17 @@ public bool InRangeOf(Entity target, int range)
             return false;
         }
 
-        public virtual void NotifySwarm(Entity attacker)
+        public virtual void NotifySwarm(Entity attacker) 
         {
+            if (MapController.TryGetInstanceFromMap(MapId, MapInstanceId, out var instance))
+            {
+                instance.GetEntities(true).ForEach(
```

Comment:
for or foreach loop (not .ForEach()) is preferred because it doesn't create a closure.